src/plugins/intel_gpu/src/graph/impls/ocl/lstm_seq.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "primitive_base.hpp"
+
+#include "lstm_seq_inst.h"
+#include "lstm/lstm_seq_kernel_selector.h"
+#include "lstm/lstm_seq_kernel_base.h"
+#include "openvino/op/lstm_sequence.hpp"
+
+namespace cldnn {
+namespace ocl {
+
+struct lstm_seq_impl : typed_primitive_impl_ocl<rnn_seq> {
```

Comment:
the impl should also be generic as well as ocl kernel

Reply:
done